src/platforms/weex/compiler/modules/props.js

```diff
@@ -22,6 +22,13 @@ function transformNode (el: ASTElement, options: CompilerOptions) {
           el.attrsMap[realName] = el.attrsMap[attr.name]
           delete el.attrsMap[attr.name]
         }
+        const rawAttrsList = el.rawAttrsList || []
+        for (let i = rawAttrsList.length - 1; i >= 0; i--) {
```

Comment:
This is a quadratic time complexity. I wonder if it will impact compile time.

Reply:
Switched to using an object to track raw attribute info in https://github.com/vuejs/vue/pull/7127/commits/59915221b2f3aafadc9aa4865b0e3fb60de64806